camptix.js

```diff
@@ -230,6 +231,24 @@ var docCookies={getItem:function(e){return decodeURIComponent(document.cookie.re
 
 }(jQuery));
 
+/**
+ * Class for utility functions
```

Comment:
I like this idea of a utility class. I think it would be good to add an example to the doc block, showing how an individual method within the class could be overridden without overriding the whole class.

Reply:
done